frontend/src/scene/main-view/ClassCard.js

```diff
@@ -388,10 +313,7 @@ class ClassCard extends React.Component {
                                 buttonLabel={buttonLabel}
                                 onButtonClick={this.selectCourse(el)}
                                 errorMessages={errorMessages}
-                                disabled={
-                                    !el.isAvailable ||
-                                    !this.props.userStore.isAuthenticated
```

Comment:
Have you check whether the app is working fine once you log out. Is the logout status shown properly once you logout?

Reply:
![Screenshot 2019-03-15 at 11 56 00](https://user-images.githubusercontent.com/9301426/54423492-75f11880-4719-11e9-9e44-184311be27fb.png)
This is how logout screen look like. 
``